.rubocop.yml

```diff
@@ -38,6 +38,10 @@ Style/NumericLiteralPrefix:
 Style/TernaryParentheses:
   Enabled: false
 
+# sometimes it is usefull to have those empty methods
```

Comment:
Here I think rubocop just wants us to do something like:

```
      def self.return_value; end
```

I am not against it. It's a few files to change.